tests/cbmc/.gitignore

```diff
@@ -7,3 +7,4 @@ logs
 property.xml
 
 .litani_cache_dir
+.ninja_log
```

Comment:
Is this necessary? What generates this?

Reply:
We're using [Litani](https://github.com/awslabs/aws-build-accumulator/) now to run all proofs, which uses ninja. So, every time we run a proof locally, we end up creating these log files. Adding it to `.gitignore` will prevent anybody pushing it by mistake.